### PR TITLE
DOC: Add to citations.bib for QIIME2

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -547,7 +547,7 @@ Empress is available in mSystems
 Empress in your work, please cite it! The BibTeX for this paper is:
 
 ```
-@article {Cantrelle01216-20,
+@article {CantrelleFedarko2021empress,
```
Comment (fedarko) [inline README.md:550]:
```suggestion
@article {CantrellFedarko2021empress,
```

File: empress/citations.bib [hunk 2]
```diff
@@ -35,4 +35,19 @@ @article{mcdonald2018unifrac
   url = {https://www.ncbi.nlm.nih.gov/pubmed/30377368},
   year = {2018},
   type = {Journal Article}
-}
\ No newline at end of file
+}
+
+@article {CantrelleFedarko2021empress,
```
Comment (fedarko) [inline empress/citations.bib:40]:
```suggestion
@article {CantrellFedarko2021empress,
```

**Review by fedarko — COMMENTED** [review]
Thanks @ElDeveloper! One small fix to @kwcantrell's name in the citation title :)

Do you think we should keep the other 3 references (Gneiss, BP, Striped UniFrac) around in `citations.bib`? Now that the paper is published (citing all 3 of these in some way), I'm not sure users of the tool necessarily need to cite all of these 3 references in addition to EMPress itself. I'm not sure what the proper etiquette here is, though :)

**Comment by ElDeveloper** [comment]
I don't think there's a citation etiquette, but I think leaving just the EMPress citation makes sense.

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empire-biplot.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F494%2Fempire-biplot.qzv), [empire.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F494%2Fempire.qzv), [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F494%2Fempress-tree.qzv), [just-fm.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F494%2Fjust-fm.qzv), [plain.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F494%2Fplain.qzv)

**Comment by fedarko** [comment]
Thanks @ElDeveloper! (and @emperor-helper :penguin: ;)

**Pull request merged**